.docksal/docksal.yml

```diff
@@ -10,63 +10,11 @@ services:
       - MIGRATE_DB_ENV
     volumes:
       - ${PROJECT_ROOT}/.docksal/etc/drush:/etc/drush:ro
-    # Expose port and let Storybook attach to it.
-    expose:
-      - 6006
-    labels:
-      - io.docksal.virtual-host=storybook.${VIRTUAL_HOST},storybook.${VIRTUAL_HOST}.*
-      - io.docksal.virtual-port=6006
 
-  # Fake email
-  mail:
```

Comment:
This is mail and solr?

Reply:
fake mail doesn't work on m chips

Reply:
grrr.  Do we need a different solution?

Reply:
i don't think we need this at this time

Reply:
Discussed with Sean and we should have mail protections on our local env.  He is making a new Docker image that should work on M chips.

Reply:
I don't know what to do here.  Created an issue: https://github.com/kanopi/drupal-starter/issues/160